community/kernel/src/main/java/org/neo4j/kernel/impl/util/collection/TimedRepository.java

```diff
@@ -117,6 +117,16 @@ public void begin( KEY key ) throws ConcurrentAccessException
         }
     }
 
+    public void removeLockSession( KEY key )
```

Comment:
Why is this method called `removeLockSession`? `TimedRepository` is a generic class.